### ISSUE TITLE
Support parsing "relay webhooks" event structures

### ISSUE THREAD

**Issue by yargevad** [SparkPost/gosparkpost#27]

**Comment by abh** [comment]
This patch doesn't work (without some extra glue) because the relay_message, uh, messages don't have a "type". I emailed support and was told this was intentional.

**Comment by yargevad** [comment]
You're right that extra glue is currently needed to parse events sent to `relay webhook` endpoints, since the current auto-parsing function calls them `Unknown` afaict.
